code/client/src/pages/Create.jsx

```diff
@@ -232,6 +272,34 @@ const Create = () => {
             </Row>
           </Space>
         </Row>
+        <Row justify='center'>
+          <Checkbox onChange={() => setDoubleOtp(!doubleOtp)}>
+            <Hint>Use two One Time Passwords for enhanced security</Hint>
```

Comment:
Should let user know what to expect - i.e. they will need to scan again on the next page. Otherwise they tick the box and nothing happens